README.md

```diff
@@ -95,7 +74,7 @@ Public CNCF TOC channel: Join the [CNCF slack](https://slack.cncf.io/) and we ar
 
 This is our [voting policy](docs/voting.md). Only TOC members can cast binding votes. (+1 Binding)
 
-## Technical Advisory Groups
+## Current Technical Advisory Groups
```

Comment:
Should this be current or previous?

Reply:
Good point!

Reply:
What about "TAG being phased out" ?

Reply:
sgtm!

Reply:
Prefer just "Technical Advisory Groups" @riaankleinhans